blueocean-rest-impl/src/main/java/io/jenkins/blueocean/service/embedded/rest/AbstractRunImpl.java

```diff
@@ -249,6 +249,15 @@ public BlueTestSummary getTestSummary() {
         return ActionProxiesImpl.getActionProxies(run.getAllActions(), this);
     }
 
+    public Collection<BlueActionProxy> getAllActions() {
```

Comment:
any reason we are not using `getActions()`? Serializing all actions is very expensive 🐛

Reply:
Removed in 5822ec4